aspnetcore/security/ip-safelist/samples/2.x/ClientIpAspNetCore/AdminSafeListMiddleware.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+
+namespace ClientIpAspNetCore
+{
+    #region snippet_ClassOnly
+    public class AdminSafeListMiddleware
+    {
+        private readonly RequestDelegate _next;
+        private readonly ILogger<AdminSafeListMiddleware> _logger;
+        private readonly string _adminSafeList;
+
+        public AdminSafeListMiddleware(RequestDelegate next, ILogger<AdminSafeListMiddleware> logger, string adminSafeList)
```

Comment:
Loose horiz scroll bars
![image](https://user-images.githubusercontent.com/3605364/45007146-4e263880-afb1-11e8-8dfc-6a356b509d81.png)

Reply:
Each line of code should be no longer than 85 characters.

Reply:
@damienbod Let me know if you want to do the breaking up of long code lines or you'd like me to do it.  The length restriction is intended to limit the need for horizontal scroll bars and applies only to code that gets displayed in the article.  Here's the current guidance on this:

>Horizontal scroll bars
>Break up long lines to avoid horizontal scroll bars. Scroll bars on code blocks make code hard to read. They're especially problematic on longer code blocks, where it may be impossible to see the scroll bar and the line you want to read at the same time.

>A good practice for minimizing horizontal scroll bars on code blocks is to break up code lines longer than 85 characters. But keep in mind that the presence or absence of a scroll bar is not the only criterion of readability. If breaking a line before 85 hurts readability in other ways, feel free to go over 85.

Reply:
@tdykstra  can you fix that?

Reply:
@damienbod I'll go ahead and do the line breaks.